airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/concurrent/adapters.py

```diff
@@ -219,7 +215,7 @@ def read(self) -> Iterable[Record]:
         try:
             for record_data in self._stream.read_records(
                 cursor_field=self._cursor_field,
-                sync_mode=SyncMode.full_refresh,
+                sync_mode=SyncMode.incremental,  # TODO: check why this was full refresh
```

Comment:
the sync mode shouldn't matter / the parameter shouldn't be used. Does changing to incremental change the behavior?

Reply:
It doesn't, it just better reflects what the actual behavior is if someone is using it for some reason. Here it was already hardcoded, so maybe it makes sense not to change it, but in other places we had to [make a decision](https://github.com/airbytehq/airbyte/pull/37022/files#diff-93127bface0b323fe43b21cdb8fb14493dd465995b085a4f81647f3697930bddR318) and thought that incremental made more sense. Was doing that here just to keep things consistent.

Reply:
makes sense. I'm not opposed to changing it. thanks for the context!

Reply:
I think that was `full_refresh` because we made this codepath available to full refresh before incremental. I probably forgot to un-hardcode it when we opened it up to incremental too. 😬